controller/src/main/java/io/pravega/controller/server/ControllerServiceConfig.java

```diff
@@ -50,6 +50,8 @@
      */
     boolean isControllerClusterListenerEnabled();
 
+    String getTlsEnabledForSegmentStore();
```

Comment:
java doc, should it return boolean ?

Reply:
Sure, have added the Java doc. 

As for making it a boolean, @shiveshr had a similar comment, to which I responded in https://github.com/pravega/pravega/pull/4163#discussion_r324076746. 

Please see if it makes sense.